tests/observability/metrics/conftest.py

```
@@ -495,6 +495,18 @@ def vm_migration_metrics_vmim_scope_class(admin_client, vm_for_migration_metrics
         yield vmim
 
 
+@pytest.fixture()
```

Ruff (0.15.10)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)